doc/modules/classification_threshold.rst

```diff
@@ -0,0 +1,166 @@
+.. currentmodule:: sklearn.model_selection
+
+.. _tunedthresholdclassifier:
+
+======================================================
+Tuning cut-off decision threshold for class prediction
+======================================================
+
```

Comment:
I would start with a sentence like:

Classification is best divided into two parts:
1. the statistical problem of learning a model to predict, ideally, class probabilities
2. the decision problem to take concrete action based on those probability predictions.

Maybe put the tumor detecting example already here. Or a simpler example with taking an umbrella based on the weather prediction.

Reply:
I rephrase a bit the first paragraph to take this suggestion into account.